b8/setup.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+sudo chown nobody file1.txt.gpg
```

Comment:
if the checkoff is going to ask what permissions the files have, how to fix them, and the contents of the files, everything is given away in this commit. Since you're already seeding a private key, perhaps have the setup script create all these from sources that are asymmetrically encrypted using the public key that we're seeding, then decrypt them to disk with the privkey and fix the permissions. e.g. a couple lines of

```bash
cat <<EOF | gpg -d --output filen.txt && chmod 400 filen.txt
-----BEGIN PGP MESSAGE-----

hQEMA8JSjeQBGoyFAQgAsQtT68LWBecXsFplfRGYkX+67Z4rkPjrc2+TmU6yJhON
ykW0WRo0MAQnP26Me15920kT8LyhtnR0q3Ts6QodaVkV3N0Y+9Arj6SYQI/rVhTD
mIetbBw3RBqywnYmprGg8Fe1sKI1OuGvmD1Fsvdk1/rg/7HXSedB+D/XWzMYML6r
sk/JXldsTka+QvcbhvmqsB1ww4tPG2uJpY+u1Tg7fgZdSxxQMYj1P8CWGRiIbRhh
xUoMYCk5qE42SJTdxiC4qGMKs4+MpcW7Ob9zrWbpHs6dJeALfdbuyqPqLfWK8Os4
pinENROR/ZqPjBjo7u71MR7iktoJVqw7/lxnqpePeNJHAdz4Qxmk2JPoaMSXLZh3
u3TvKd/+Wa9ayjbBbXnnPv9CXxYmTu2YU80NlB4vA1eXpfFhBr4dtzEZcuoNcuq+
h/bonZ8a16A=
=hmQJ
-----END PGP MESSAGE-----
EOF
```